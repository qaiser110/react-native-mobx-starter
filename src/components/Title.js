import React from 'react'
import { Header, Title, Button, Icon, Left, Right, Body } from 'native-base'

export default ({ text }) => (
  <Header>
    <Body>
      <Title>{text}</Title>
    </Body>
    <Right />
  </Header>
)
