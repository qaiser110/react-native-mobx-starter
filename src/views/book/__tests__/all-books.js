import React from 'react'
import { shallow } from 'enzyme'
import AllView from '../tabs/all-books'
import Title from '../../../components/Title'

it('renders correctly', () => {
  const wrapper = shallow(<AllView />)
  expect(wrapper).toMatchSnapshot()

  expect(wrapper.find(Title)).toBeTruthy()
  expect(wrapper.find(Title).dive().length).toBe(1)
})
