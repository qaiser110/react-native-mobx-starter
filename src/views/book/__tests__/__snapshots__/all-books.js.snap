// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Styled(Container)
  style={
    Object {
      "backgroundColor": "#FFF",
    }
  }
>
  <Component
    text="All Books"
  />
  <BookListView
    filter="All"
  />
</Styled(Container)>
`;
